packages/sslconf/sslconf.0.8.3/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "Tony Wuersch <tony.wuersch@gmail.com>"
+homepage: "https://github.com/awuersch/sslconf"
+dev-repo: "https://github.com/awuersch/sslconf.git"
+bug-reports: "https://github.com/awuersch/sslconf/issues"
+doc: "https://awuersch.github.io/sslconf/doc"
+authors: [
+  "Tony Wuersch <tony.wuersch@gmail.com>"
+]
+license: "ISC"
+depends: [
+  "jbuilder" {build}
+  "ppx_sexp_conv" {build}
+  "sexplib" {build}
+  "astring" {build}
+  "rresult" {build}
+  "fpath" {build}
+  "cmdliner" {build}
+  "topkg-jbuilder" {build}
```

Comment:
According to what I read [here](https://github.com/awuersch/sslconf/blob/12dcedeb3f2f6caedd68c3e5d8fc091c4f043db8/lib/jbuild#L7) there are too many things that are `build` deps here. 

To clarify: you should have `{build}` deps for what is needed in order to build the package (here `jbuilder` and `topkg-jbuilder`). `{test}` deps for what is nedeed to build the tests of the package (here `ounit`). All the rest seems to be regular dependencies needed by the libraries and binaries you install and should not be labelled in any way.

Reply:
Regarding `ppx_sexp_conv` I think it should be a `build` dependency aswell, however you should add `sexplib` to the list of deps.

Reply:
(sorry, I could have spotted that by having a look first)

Reply:
No problem. You are already doing a lot.

Reply:
@dbuenzli , build errored when I made ounit {test} in opam. The test/jbuild had sslconf_test and sslconf_dumpcases defined with 'public_names'. I can remove the public_names line in test/jbuild. Should that fix it? The other edits, no problem.

Reply:
It depends whether you want to install these tools or not for the end user.

Reply:
Just to be clear if you install [these things](https://github.com/awuersch/sslconf/blob/12dcedeb3f2f6caedd68c3e5d8fc091c4f043db8/test/jbuild#L6) for the end user then none of these dependencies can be a `{test}` dependency.

Reply:
(and if you don't install them then all of these can become `{test}` dependencies).

Reply:
Clear, good. The end user doesn't need them. Thanks.

Reply:
(I'll make these changes tomorrow.)